shell.nix

```diff
@@ -4,9 +4,15 @@
 
 with pkgs;
 let
+  unstable = import
+  (fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/nixpkgs-unstable.tar.gz)
```

Comment:
Why is this floating import necessary?

Reply:
That's a good point. Probably better to lock it to a particular commit.